### DISCUSSION TITLE
Zoom from non-children?

### DISCUSSION THREAD

**Discussion by ianthedev** [dimsemenov/PhotoSwipe#1996] [Q&A] [answered]
My use case is like this: I'm using the [center mode of a carousel plugin](https://splidejs.com/#focus-center-heading) for my client's photo gallery, and clicking any carousel item should zoom PhotoSwipe. By the way, the "zoom" here refers to `showHideAnimationType`.

Because the center mode of the carousel plugin creates clones of the original carousel items, I have to define `children` of PhotoSwipe as only the original carousel items so that PhotoSwipe would not count cloned carousel items in. That caused a problem: clicking cloned carousel items does not zoom PhotoSwipe.

So what I need to do is: when users click a cloned carousel item, I need to programmatically zoom PhotoSwip from that cloned carousel item.

Any advice would be appreciated.

**Comment by dimsemenov** [comment, marked as answer]
You'll prob need to separate dom and data https://photoswipe.com/data-sources/#separate-dom-and-data or use CSS selector that doesn't involve clones.
